src/lib.rs

```diff
@@ -35,6 +35,7 @@ pub fn fix_vars(vars: &[&str]) -> std::result::Result<(), Error> {
     let shell = std::env::var("SHELL").unwrap_or_else(|_| default_shell.into());
 
     let out = std::process::Command::new(shell)
+      .current_dir(home::home_dir().unwrap_or_default())
```

Comment:
i don't think we should .unwrap_or_default() here, but only change the CWD if we can find a home dir

Reply:
Done. Good point :+1: